scripts/examples/simple_tcp_server.py

```diff
@@ -5,7 +5,7 @@
 from collections import defaultdict
 
 from umodbus import conf
-from umodbus.server import get_server
+from umodbus.server.tcp import get_server
 from umodbus.server.tcp import RequestHandler
```

Comment:
Ok, done. 
Travis reports code is broken in pypy3 but I don't think it's because of me